tests/kubelet-service-kill_test.go

```diff
@@ -87,6 +87,19 @@ var _ = Describe("BDD of kubelet-service-kill experiment", func() {
 
 		It("Should check for creation of runner pod", func() {
 
+			//Get a single node name
+			nodes, err := client.CoreV1().Nodes().List(metav1.ListOptions{})
+			Expect(err).To(BeNil(), "fail to get the nodes")
+			nodeName := nodes.Items[0].Name
+
+			//Cordon node
+			By("Cordoning node")
+			err = exec.Command("kubectl", "cordon", nodeName).Run()
+			Expect(err).To(BeNil(), "failed to cordon the application node")
```

Comment:
I'm trying to understand why did we cordon 0th node and how it is the application node?

Reply:
ok, are assuming that we have 3 replicas on three nodes?

Reply:
In chaos engine env's we are setting the same node as the value of APP_NODE (in other experiments node-drain,node-taint we are using same approach in e2e)

Reply:
ok got it

Reply:
As now, we are passing APP_NODE as an explicit variable so even if all replicas are on the same node. it won't be our concern but earlier we have that doubt in our mind when we are deriving app_node instead of passing it as env